packages/components/pie-textarea/test/component/pie-textarea.spec.ts

```diff
@@ -320,6 +320,25 @@ test.describe('PieTextarea - Component tests', () => {
             });
         });
 
+        test.describe('defaultValue', () => {
+            test('should correctly reset the textarea value to the default value if one is provided when the form is reset', async ({ page }) => {
```

Comment:
Do you need a test for just the default value?

`should set the "defaultValue" when a default value is used`

Reply:
`defaultValue` is an optional default value to use when the textarea is reset.

In my opinion this test covers this case, because: 
1. Set the `defaultValue`
2. Type a `value`
3. Reset form
4. Get a `defaultValue` as it was expected

Reply:
I moved this test from "Props" test group to "Form integration"